lib/pageflow/global_config_api.rb

```diff
@@ -121,6 +121,14 @@ def build_config(&block)
       end
     end
 
+    def call_entry_type_config_blocks(config, target)
+      type_name = target.type_name
+      delegator = ConfigDelegator.new(config, type_name)
+      delegator.entry_type_configure_blocks[type_name].each do |block|
+        block.call(delegator)
+      end
+    end
```

Comment:
This makes all config options available inside `for_entry_type` blocks, which we don't want. There could be a test which makes sure that things like `config.mailer_sender` are not available in these blocks.